packages/apps/graph/src/devnet/simulation/simulate.ts

```diff
@@ -62,7 +62,7 @@ export async function simulate({
     for (let i = 0; i < numberOfAccounts; i++) {
       // This will determine if the account has 1 or 2 keys (even = 1 key, odd = 2 keys)
       const noOfKeys = i % 2 === 0 ? 1 : 2;
-      let account = await generateAccount(noOfKeys);
+      const account = await generateAccount(noOfKeys);
```

Comment:
I think this will bring an issue, because when we're funding accounts, in the case of a cross-chain transfer, we alter the account chain, to chain 1. Afterwards, we save the account on `accounts` array which is used to keep track of the existing accounts (with funds). If this is a constant and we do not register that the funded chain was chain 1 and not chain 0, we will run into an error when we try to transfer funds from that account.
![image](https://github.com/kadena-community/kadena.js/assets/112877112/06d6f929-0e27-4db3-9a9b-de9cb2a8e7c6)

Reply:
fixed